transifex/native/tools/migrations/review.py

```diff
@@ -296,12 +297,15 @@ def _yes_no(self, description, yes_message=None, no_message=None):
 
 class NoopReviewPolicy(ReviewPolicy):
     """Never prompts the user for anything."""
-    pass
+
+    name = 'none'
```

Comment:
Again consider capitalising and protecting name. Also, you should add the name attribute to the 'abstract' ReviewPolicy class